cmd/clusterctl/client/repository/repository_local.go

```diff
@@ -144,10 +144,14 @@ func newLocalRepository(providerConfig config.Provider, configVariablesClient co
 	// gets the path part of the url and check it is an absolute path
 	// in case of windows, we should take care of removing the additional / which is required by the URI standard
 	// for windows local paths. see https://blogs.msdn.microsoft.com/ie/2006/12/06/file-uris-in-windows/ for more details
-	path := url.EscapedPath()
+	path := url.Path
 	if runtime.GOOS == "windows" {
 		path = strings.TrimPrefix(path, "/")
 		path = filepath.FromSlash(path)
+	} else {
+		// Windows 10 update 1803 stops decoding file paths
+		// https://support.microsoft.com/en-us/help/4467268/url-encoded-unc-paths-not-url-decoded-in-windows-10-version-1803-later
```

Comment:
ideally, the note about Windows should be in the Windows branch above, indicating that escaping is not required for older versions and not supported in 1803 and later, linking to the page about the UNC change.

the "in case of windows..." note should be moved in the Windows branch as well.

Reply:
Moved both comments within `if` block